platform/core/PolyPodCoreAndroid/core/src/main/kotlin/coop/polypoly/core/UpdateNotification.kt

```diff
@@ -0,0 +1,21 @@
+package coop.polypoly.core
+
+class UpdateNotification {
+    companion object {
+        val showInApp get() = Core.executeRequest(
+            CoreRequest.GetShowInAppNotification(),
+            Boolean::fromValue
+        )
+
+        val showPush get() = Core.executeRequest(
+            CoreRequest.GetShowPushNotification(),
+            Boolean::fromValue
+        )
+
+        fun handlePushSeen() =
+            Core.executeRequest(CoreRequest.HandlePushNotificationSeen())
+
+        fun handleInAppSeen() =
+            Core.executeRequest(CoreRequest.HandleInAppNotificationSeen())
+    }
+}
```

Comment:
This UpdateNotification.kt can be used to group together CoreRequests that relate to UpdateNotification, since Core.executeRequest accepts any request.

Reply:
It's a bit more boilerplate, but I personally found it a bit nicer to deal with. We could refactor it away without much trouble though.